shopify/loaders/ProductListingPage.ts

```diff
@@ -73,7 +73,7 @@ const loader = async (
 
   const count = props.count ?? 12;
   const query = props.query || url.searchParams.get("q") || "";
-  const page = props.page || Number(url.searchParams.get("page")) || 0;
+  const page = props.page || Number(url.searchParams.get("page")) || 1;
```

Comment:
I think we should bring it optional. Did you see how other integrations are doing that? pageOffSet prop

Reply:
thanks, @guitavano.